resources/devbox/macos/macos.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+## Note: does not use standard build script include
+# adjust relative paths as necessary. Note that this will not work in a symlinked path reliably
+THIS_SCRIPT="${BASH_SOURCE[0]}"
+THIS_DIR=$(dirname "$THIS_SCRIPT")
+
+# Determine which components to install
+REQUIRE_ANDROID=false
+REQUIRE_IOS=false
+REQUIRE_MACOS=false
+REQUIRE_WEB=false
+
+# Optional components
+REQUIRE_KMCOMP=false
+REQUIRE_PANDOC=false
+REQUIRE_SENTRYCLI=false
+
+# Parse args
+shopt -s nocasematch
+
+while [[ $# -gt 0 ]] ; do
+    key="$1"
+    case $key in
+        --help)
+            echo "Usage: macos.sh targets"
+            echo "  targets: one or more of: "
+            echo "             android ios macos web all"
+            echo "  optional targets:"
+            echo "             kmcomp pandoc sentry-cli"
+            ;;
+        android)
+            $REQUIRE_ANDROID=true
+            ;;
+        ios)
+            $REQUIRE_IOS=true
+            ;;
+        macos)
+            $REQUIRE_MACOS=true
+            ;;
+        web)
+            $REQUIRE_WEB=true
+            ;;
+        kmcomp)
+            $REQUIRE_KMCOMP=true
+            ;;
+        pandoc)
+            $REQUIRE_PANDOC=true
+            ;;
+        sentry-cli)
+            $REQUIRE_SENTRYCLI=true
+            ;;
+        all)
+            REQUIRE_ANDROID=true
+            REQUIRE_IOS=true
+            REQUIRE_MACOS=true
+            REQUIRE_WEB=true
+            ;;
+        *)
+            echo "Error: unrecognised parameter. Use --help for help"
+            ;;
+    esac
+    shift
+done
+
+# This script will configure your development environment from a bare metal mac install. It should be idempotent.
+
+echo "This script will configure your macOS computer to build Keyman, installing build tools and prerequisites."
+echo "You can also do this yourself following the notes in building.md."
```

Comment:
`building.md`? Where is that?

Reply:
Yes this should probably do the same as the android readme.md etc and have 
```See [build configuration](../docs/build/index.md) for details on how to configure your build environment.```
Or say see docs/buil/macos-note.md for configure build environment and macos.md for building.

Reply:
```suggestion
echo "You can also do this yourself following the notes in docs/build/macos.md."
```